examples/encoding/create_simple_encoding_hls_with_multiple_audio_groups_and_stream_conditions/main.go

```diff
@@ -0,0 +1,360 @@
+
+	aacS := services.NewAACCodecConfigurationService(bitmovin)
+	aacConfig128k := &models.AACCodecConfiguration{
+		Name:         stringToPtr("example_audio_codec_configuration"),
+		Bitrate:      intToPtr(128000),
+		SamplingRate: floatToPtr(48000.0),
+	}
+	aac128kResp, err := aacS.Create(aacConfig128k)
+	errorHandler(aac128kResp.Status, err)
+
+	aacConfig98k := &models.AACCodecConfiguration{
+		Name:         stringToPtr("example_audio_codec_configuration"),
+		Bitrate:      intToPtr(98000),
+		SamplingRate: floatToPtr(48000.0),
+	}
+	aac98kResp, err := aacS.Create(aacConfig98k)
+	errorHandler(aac98kResp.Status, err)
+
+	videoInputStream := models.InputStream{
+		InputID:       httpResp.Data.Result.ID,
+		InputPath:     stringToPtr("YOUR INPUT FILE PATH AND LOCATION"),
+		SelectionMode: bitmovintypes.SelectionModeAuto,
+	}
+	audioInputStream := models.InputStream{
+		InputID:       httpResp.Data.Result.ID,
+		InputPath:     stringToPtr("YOUR INPUT FILE PATH AND LOCATION"),
+		SelectionMode: bitmovintypes.SelectionModeAuto,
+	}
+
+	vis := []models.InputStream{videoInputStream}
+
+	videoStream1080p := &models.Stream{
+		CodecConfigurationID: video1080pResp.Data.Result.ID,
+		InputStreams:         vis,
+	}
+
+	videoStream720p := &models.Stream{
+		CodecConfigurationID: video720Resp.Data.Result.ID,
+		InputStreams:         vis,
+	}
+
+	videoStream1080pResp, err := encodingS.AddStream(*encodingResp.Data.Result.ID, videoStream1080p)
+	errorHandler(videoStream1080pResp.Status, err)
+	videoStream720pResp, err := encodingS.AddStream(*encodingResp.Data.Result.ID, videoStream720p)
+	errorHandler(videoStream720pResp.Status, err)
+
+	ais := []models.InputStream{audioInputStream}
+	audioStream128k := &models.Stream{
+		CodecConfigurationID: aac128kResp.Data.Result.ID,
+		InputStreams:         ais,
+		Conditions:           models.NewAttributeCondition(bitmovintypes.ConditionAttributeBitrate, ">=", "120000"),
+	}
+	aacStream128kResp, err := encodingS.AddStream(*encodingResp.Data.Result.ID, audioStream128k)
+	errorHandler(aacStream128kResp.Status, err)
+
+	audioStream98k := &models.Stream{
+		CodecConfigurationID: aac98kResp.Data.Result.ID,
+		InputStreams:         ais,
+	}
+	aacStream98kResp, err := encodingS.AddStream(*encodingResp.Data.Result.ID, audioStream98k)
+	errorHandler(aacStream98kResp.Status, err)
+
+	aclEntry := models.ACLItem{
+		Permission: bitmovintypes.ACLPermissionPublicRead,
+	}
+	acl := []models.ACLItem{aclEntry}
+
+	videoMuxingStream1080p := models.StreamItem{
+		StreamID: videoStream1080pResp.Data.Result.ID,
+	}
+	videoMuxingStream720p := models.StreamItem{
+		StreamID: videoStream720pResp.Data.Result.ID,
+	}
+	audioMuxing128kStream := models.StreamItem{
+		StreamID: aacStream128kResp.Data.Result.ID,
+	}
+	audioMuxing98kStream := models.StreamItem{
+		StreamID: aacStream98kResp.Data.Result.ID,
+	}
+
+	videoMuxing1080pOutput := models.Output{
+		OutputID:   s3OutputResp.Data.Result.ID,
+		OutputPath: stringToPtr("golang_test/video/1080p"),
```

Comment:
Maybe define the baseOutputPath as a variable at the beginning and only change the relative paths here
e.g.:
```
	outputBasePath := fmt.Sprintf("your/output/base")
```
```
	videoMuxing1080pOutput := models.Output{
		OutputID:   s3OutputResp.Data.Result.ID,
		OutputPath: stringToPtr(fmt.Sprintf("%s/video/1080p", outputBasePath)),
		ACL:        acl,
	}
```